### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by NGYLK** [MIC-DKFZ/nnDetection#301]
### :question: Question

Hello, first of all, thank you for your contributions. I encountered some minor issues while training nnDetection. The first is the ​**"Proxy FG Dice: 0.791"** value in the training logs. Is this the segmentation accuracy (DICE)? To my knowledge, the model outputs detection boxes instead of segmentation masks during testing. ​What does this value mean? Can it be considered as segmentation accuracy? If so, can the model output segmentation masks instead of detection boxes? ​Where in the code is this configured?

The second question is about evaluation metrics. In academic papers, common metrics like ​accuracy, sensitivity/specificity, and AUC are often used. ​Can the code be modified to output these metrics? If yes, which part of the code should be modified?

**Comment by andres2631996** [comment]
Hello,

Thanks for your question. The Proxy FG Dice in training would account for the segmentation accuracy of the segmenter in the RetinaUNet at a patch-based label, for those patches sampled by the validation dataloader. Hence, it should not be considered the Dice score in the validation set, since in that case, the different predictions per patch would need to be merged, and that would provide a different per case result. Since a Dice-based loss function is used to train the segmenter head, the Proxy Dice is logged during the training process, but it should not be taken as an actual validation metric, but rather as a guide of whether the model is learning or not, together with the classification and regression loss values that are outputted as well during training. 

nnDetection only outputs box results after post processing the different per patch predictions.

Case-level metrics such as the AUC (considering the highest confidence score for all the cases in an image) and F1, precision, recall, and accuracy (for different confidence score thresholds) can be obtained by including the flag "--case" when running "nndet eval", as for example:

`nndet_eval TASKID MODEL_FOLDER FOLD --boxes --analyze_boxes --case`

Where TASKID is your task ID, MODEL_FOLDER your model directory name and FOLD your cross-validation fold index (-1 if you are working with the consolidated results)
